content/posts/glazier.md

```diff
@@ -97,7 +97,7 @@ In order to achieve this, you'll need to do the following steps on **each device
       d-----         6/19/2021   5:01 PM                detectionverificationdrv.inf_amd64_dcc202b5af4a34b5
       ...
       ```
-1. Repeat the above steps 1-3 above on each device model that you want to support.
+1. Repeat the above steps 1-3 above on each device model that you want to support, keeping a separate folder for each model. Reference the [total control](https://www.deploymentresearch.com/mdt-2013-lite-touch-driver-management/) method for driver management for more information.
```

Comment:
> Repeat the above steps 1-3 above on each device model that you want to support, keeping a separate folder for each model

If I end up supporting more complex subfolder structures in `bootstrap_winpe.ps1` this suggestion is definitely an improvement.

> Reference the [total control](https://www.deploymentresearch.com/mdt-2013-lite-touch-driver-management/) method for driver management for more information

This reference to MDT confuses me though. I'm not using MDT at all and the link adds good context.

Reply:
The aim here is just to direct the reader for more information around why it's important to have subfolders for drivers. I'll leave it up to you if you feel it's necessary.